src/data/community-events.json

```
@@ -385,6 +385,15 @@
     "description": "We are building both a hub for regional projects with big ambitions; and, a gateway for local and international brands to connect with the Ethereum Brasil builder community.",
     "startDate": "2022-09-09",
     "endDate": "2022-09-11"
+  },
+    {
+    "title": "Arab Blockchain Week 2022",
+    "to": "https://www.arabblockchainweek.com/",
+    "sponsor": null,
+    "location": "Hybrid ( online (Livaatverse) & in-persion Dubai, Cairo & Amman",
```

We can't support this text here. I'll just leave it as Hybrid for now.

```suggestion
    "location": "Hybrid",
```